docs/day-2-operations/Backups.md

```diff
@@ -0,0 +1,272 @@
+
+Also, some parts of the data are synchronized between different components. One copy is considered the "source of truth" for the data set, and the other a copy that is not modified locally, just synchronized from the "source of truth". In these cases, upon restore, the "source of truth" should be restored and then the copies in other components synchronized from it.
+
+## Data Sets
+This section goes into more detail on the different data sets in the different persistent stores, their purposes, the storage type used and if it is the "source of truth" or not.
+
+The full state of a 3scale deployment is stored across these services and their persistent volumes:
+
+* `system-mysql`:  Mysql database (`mysql-storage`)
+* `system-storage`: Volume for Files
+* `zync-database`: Postgres database for `zync` component. This uses "HostPath" as storage. If the pod is moved into another node the data is lost. That is not a problem because the data are sync jobs and does not need to be 100% persistent.
+* `backend-redis`: Redis database (`backend-redis-storage`)
+* `system-redis`: Redis database (`system-redis-storage`)
+
+### `system-mysql`
+This is a relational database for storing the information about users, accounts, APIs, plans etc in the 3scale Admin Console. 
+
+A subset of this information related to Services is synchronized to the `Backend` component and stored in `backend-redis`. `system-mysql` is the source of truth for this information. 
+
+### `system-storage`
+This is for storing files to be read and written by the `System` component. They fall into two categories:
+
+* Configuration files read by the System component at run-time
+* Static files (HTML, CSS, JS, etc) uploaded to System by it's CMS feature, for the purpose of creating a Developer Portal.
+
+Note that `System` can be scaled horizontally with multiple pods uploading and reading said static files, hence the need for a `RWX` PersistentVolume.
+
+### `zync-database`
+This is a relational database for storing information related to the synchronization of identities between 3scale and an Identity provider.
+This information is not duplicated in other components and this is the sole source of truth.
+
+### `backend-redis`
+This contains multiple data sets used by the `Backend` component:
+
+* Usages: This is API usage information aggregated by `Backend`. It is used by `Backend` for rate-limiting decisions and by `System` to display analytics information in the UI or via API.
+* Config: This is configuration information about Services, Rate-limits, etc that is synchronized from `System` via an internal API. This is NOT the source of truth of this info, `System` and `system-mysql` is.
+* AuthKeys: Storage of OAuth keys created directly in `Backend`. This is the source of truth for this information.
+* Queues: Queues of Background Jobs to be executed by worker processses. These are ephemeral and are deleted once processed.
+
```

Comment:
There are others like alerts and integration errors. Backend is also the source of truth for those.
In the past we had "latest transactions" as well, but I think that's deprecated now.

Reply:
Understood. Good points.

But overall recommendation (I think) is:
- restore it
- then sync from system which will over-write some info, others not

and that would take care of the other cases you point out @davidor ?

Reply:
Add here the missing ones...

Reply:
@davidor could you ad the missing ones as @andrewdavidmackenzie requested?

Reply:
Issue created to address the split of the backend databases: https://issues.jboss.org/browse/THREESCALE-2745